### PR TITLE
Added pix2pix interactive example

### PR DESCRIPTION
**Pull request by yining1023**
This PR adds a pix2pix interactive example that look like this:
screen shot 2018-06-15 at 7 45 00 pm

Does the text on the example make sense?
I'm also going to write a tutorial about [how to prepate the data and train your own pix2pix model](https://github.com/yining1023/pix2pix_edge2pikachu_deeplearnjs_old/blob/master/README.md).

### CODE REVIEW

**Comment by cvalenzuela** [comment]
Nice! This will probably get merged after https://github.com/ml5js/ml5-library/pull/162 gets merged

File: p5js/Pix2Pix_callback/sketch.js [hunk 1]
```diff
@@ -61,13 +67,17 @@ function transfer() {
 
   // Select canvas DOM element
   let canvasElement = document.getElementById('defaultCanvas0');
```
Comment (shiffman) [inline p5js/Pix2Pix_callback/sketch.js]: I might suggest using p5.dom for the p5 examples? Would `inputCanvas.elt` get you the element you are looking for or no? In general I would avoid using `'defaultCanvas0'` in code and explicitly assign an `id` to be more clear.

File: p5js/Pix2Pix_callback/sketch.js [hunk 2]
```diff
@@ -61,13 +67,17 @@ function transfer() {
...
+
   // Apply pix2pix transformation
-  pix2pix.transfer(canvasElement, function(result) {
-    // Clear output container
-    outputContainer.html('');
-    // Create an image based result
-    createImg(result.src).class('border-box').parent('output');
+  pix2pix.transfer(canvasElement, function(err, result) {
+    if (err) console.log(err);
```
Comment (shiffman) [inline p5js/Pix2Pix_callback/sketch.js]: For p5 examples it's generally preferable to always use `{}` even when they can be left out.

**Comment by yining1023** [comment]
Thank you!

I changed it to
```javascript
const canvasElement = select('canvas').elt;
```

We support p5 video element, maybe we should support p5 canvas element too. So in the future, people can just pass in 
```javascript
const p5Canvas = select('canvas');
pix2pix.transfer(p5Canvas, function(err, result) {})
```

**Comment by cvalenzuela** [comment]
I'm going to merge this for now, we can add support for p5 canvas later in a separate PR in the library!

**Pull request merged**